Makefile

```
@@ -45,11 +45,10 @@ ci-non-test: verify licenses vulncheck ## Runs checks other than tests
 run: ## Run Karpenter controller binary against your local cluster
 	SYSTEM_NAMESPACE=${KARPENTER_NAMESPACE} \
 		KUBERNETES_MIN_VERSION="1.19.0-0" \
-		LEADER_ELECT=false \
+		DISABLE_LEADER_ELECT=true \
```

```suggestion
		DISABLE_LEADER_ELECTION=true \
```

I'd rename this to `DISABLE_LEADER_ELECTION` instead of leader elect